scholia/app/templates/work_statements.sparql

```diff
@@ -17,6 +17,7 @@ WITH {
     ?item ?p ?statement .
     ?property wikibase:claim ?p . 
     ?statement ?a ?value .
+    FILTER (!CONTAINS(LCASE(STR(?a)), "http://www.wikidata.org/prop/qualifier/"))
```

Comment:
Wouldn't STRSTARTS be better? 

https://www.w3.org/TR/sparql11-query/#func-strstarts

Reply:
Yes, that's better. Fixed via https://github.com/WDscholia/scholia/pull/2047/commits/b10ef54bb48ce3a06e3a17d34685f38445cb7ef4 .